items/item.go

```diff
@@ -34,6 +37,12 @@ func (s *Store) SetCache(cache ItemCache) {
 	s.cache = cache
 }
 
+// For togging tape_only mode - see  DLTP-520
```

Comment:
Make this comment conform to the other documentation: Start with "SetUseStore enables or disables access to the underlying store." Then indicate what each possible value for `value` means.